src/main/scala/viper/gobra/reporting/VerifierError.scala

```diff
@@ -266,6 +266,11 @@ case class ChannelMakePreconditionError(info: Source.Verifier.Info) extends Veri
   override def localMessage: String = s"The provided length to ${info.origin.tag.trim} might be negative"
 }
 
+case class RangeVariableMightNotExistError(info: Source.Verifier.Info)(varName: String, rangeExpr: String) extends VerificationError {
+  override def localId: String = "range_variable_might_not_exist"
+  override def localMessage: String = s"Range variable '$varName' might not exist if the length of expression '$rangeExpr' is 0"
```

Comment:
```suggestion
  override def localMessage: String = s"Range variable '$varName' might not have been declared because the length of the expression '$rangeExpr' might be 0"
```